web/src/components/home/subcomponents/welcome/messages.js

```diff
@@ -0,0 +1,12 @@
+const messages = {
+  discover: "Discover your rhythm at",
+  title: "Do Your Own Swing",
+  classesAndSocial:
+    "West Coast Swing classes and social dancing in San\u00A0Jose,\u00A0CA.",
+  consciousSpace:
+    "A COVID-Conscious, Queer-Led, 2SLGBTQIA+ Affirming\u00A0Space",
```

Comment:
Try using `text-wrap: balance` instead

Also please add the rainbow icon at the end of affirming space

Reply:
Now it sort of breaks only in 1 spot, which may look unbalanced on specific screen sizes, but we should be okay